sw/device/lib/dif/dif_clkmgr_unittest.cc

```diff
@@ -296,5 +296,39 @@ TEST_F(HintableClockTest, GetEnabledError) {
             kDifBadArg);
 }
 
+class MeasureCtrlTest : public ClkMgrTest {};
+
+TEST_F(MeasureCtrlTest, Disable) {
+  EXPECT_WRITE32(CLKMGR_MEASURE_CTRL_REGWEN_REG_OFFSET, 0);
+  EXPECT_EQ(dif_clkmgr_measure_ctrl_disable(&clkmgr_), kDifOk);
+}
+
+TEST_F(MeasureCtrlTest, DisableError) {
+  // Null handle.
+  EXPECT_EQ(dif_clkmgr_measure_ctrl_disable(nullptr), kDifBadArg);
+}
+
+TEST_F(MeasureCtrlTest, GetEnable) {
+  {  // enabled
```

Comment:
Maybe these aught to be separate tests, since they share no state?

Reply:
Tests like lines 203 and 251 have a similar pattern, so to be consistent I'd rather keep this.